PyPrograms/contacts/contacts.py

```diff
@@ -1,42 +1,48 @@
-# project by Jym Patel
-# it uses GPL v3 License
+# code by @JymPatel
+# edited by (editors can put their name here && thanks for contribution :)
 
+# this code uses GPL V3 LICENSE
+print("this code uses GPL V3 LICENSE")
+print("")
+
+# start of code
 # defining array
 fname = ["Jym"]
 lname = ["Patel"]
 number = [""]
 email = ["jympatel@yahoo.com"]
-    
+
 print("CLOSING PROGRAM WILL RESET ALL CHANGES MADE IN YOUR CONTACT LIST\n##")
```

Comment:
Perhaps we can use the python module pickle to store data? Its a bit hard to use but it would work.

Reply:
I don’t know about pickle, I just know basics. I need time to work on Pickle

Reply:
can you add pickle in this update, if you have time to do so.

Reply:
next update. I still got to learn pickle.

Reply:
Ok. That’s fine, I also wanted pickle on next update